### ISSUE TITLE
WorldEdit just stoppped working at one point

### ISSUE THREAD

**Issue by Saladdddd** [EngineHub/WorldEdit#1569]
**Versions**

WorldEdit version:mc1.16.3-7.2.0-rc-01
Platform version:1.16.3

**Describe the bug**

I dont have any command feedback in the chat, every other command works fine. no commands even work, axe does behave like a normal axe
**To Reproduce**

1. ...
2. ...

**Expected behavior**

every command that worldedit has like //help or //pos doesnt work
**Screenshots**

**Additional context**

[error code.txt](https://github.com/EngineHub/WorldEdit/files/5412366/error.code.txt)

**Comment by wizjany** [comment]
use a dev build

**Comment by Saladdddd** [comment]
> use a dev build

the thing is, i had that version the whole time and i have a second server with the same build working completely fine

**Comment by wizjany** [comment]
idk what "that version" or "same build" mean. rc1 has a bug that current dev builds (and even previous versions like beta5) don't.

**Comment by Saladdddd** [comment]
then let me rephrase it, ive been using rc1 since you guys were providing it and ive been using it without problem, just at some point it randomly stopped

**Comment by wizjany** [comment]
yes. you can also delete the `worldedit-unpack` folder in your `java.io.tmpdir` if you want, just much easier to update imo.

**Comment by Saladdddd** [comment]
thank you for a answer that isnt just a "just dont use it"

**Comment by wizjany** [comment]
you weren't asking, you reported a bug and I marked it invalid since it's not a bug in the current version

**Comment by Saladdddd** [comment]
than at least say that

**Comment by wizjany** [comment]
I did? I said using a dev build was the solution here? idk why you're making such a fuss over it.
